apps/shell/app.js

```diff
@@ -48,54 +145,30 @@ wss.on('connection', function connection (ws, req) {
   var match;
   var host = process.env.DEFAULT_SSHHOST || 'localhost';
   var dir;
-  var term;
-  var cmd, args;
-
+  var extraction = regexPathMatch.exec(req.url);
+  var uuid = extraction[0];
+  
   console.log('Connection established');
 
+
+
   // Determine host and dir from request URL
-  if (match = req.url.match(process.env.PASSENGER_BASE_URI + '/ssh/([^\\/]+)(.+)?$')) {
+  if (match = req.url.match(process.env.PASSENGER_BASE_URI + `/session/${uuid}([^\\/]+)(.+)?$`)) {
```

Comment:
I didn't test it yet, but from the look at it, it would seem that we would need a slash between `${uuid}` and `([^\\/]+)` i.e. `/session/${uuid}/([^\\/]+)(.+)?$`.